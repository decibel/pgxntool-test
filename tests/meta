#!/bin/bash

trap 'echo "ERROR: $BASH_SOURCE: line $LINENO" >&2' ERR
set -o errexit -o errtrace -o pipefail
#set -o xtrace -o verbose

if [ "$1" == "-v" ]; then
  verboseout=1
  shift
fi
BASEDIR=`cd ${0%/*}; pwd`

. $BASEDIR/../.env
. $TOPDIR/lib.sh
cd $TEST_REPO

DISTRIBUTION_NAME=distribution_test
EXTENSION_NAME=pgxntool-test # TODO: rename to something less likely to conflict

out Verify changing META.in.json works
# Need to sleep 1 second otherwise make won't pickup new timestamp
sleep 1
# Sanity check
[ -n "$DISTRIBUTION_NAME" ] && [ -n "$EXTENSION_NAME" ]

sed -i '' -e "s/DISTRIBUTION_NAME/$DISTRIBUTION_NAME/" -e "s/EXTENSION_NAME/$EXTENSION_NAME/" META.in.json
#git diff -u

out -v This make will produce a bogus '"already up to date"' message for some reason
make
git commit -am "Change META"

check_log

# vi: expandtab sw=2 ts=2
