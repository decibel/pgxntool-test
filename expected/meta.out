# Verify changing META.in.json works
######################################
# This make will produce a bogus "already up to date" message for some reason
######################################
make[1]: `META.json' is up to date.
@GIT COMMIT@ Change META
 2 files changed, 4 insertions(+), 4 deletions(-)
